docs/core/docker/visual-studio-tools-for-docker.md

```diff
@@ -19,8 +19,8 @@ Developing and debugging your application in a Docker container can be a ceremon
 >The current version targets Linux Docker containers, with Windows Containers coming soon.
 
 ## Prerequisites
-- [Microsoft Visual Studio 2015 Update 3](https://www.visualstudio.com/downloads/download-visual-studio-vs)
-- [.NET Core 1.0.1 - VS 2015 Tooling Preview 2](https://go.microsoft.com/fwlink/?LinkID=827546)
+- [Microsoft Visual Studio 2017](https://www.visualstudio.com/)
```

Comment:
So @GuardRex, these instructions need to be reviewed. You no longer download the tool, you get it with the workloads (I believe it comes with the .NET Core workload too, need to check)